### PR TITLE
Feature/improve dockerfile

### PR DESCRIPTION
**Pull request by paullatzelsperger**
Improves the dockerfile to use an `nginx.conf` file to fix a bug where reload (Ctrl+F5) does not work.

### CODE REVIEW

File: deployment/nginx.conf [hunk 1]
```diff
@@ -1,15 +0,0 @@
-events {}
-http {
-  include /etc/nginx/mime.types;
```
Comment (marcgs) [inline deployment/nginx.conf:3]: This `nginx.conf` solved the reload issue as well. I don't understand what impact is are the changes in the new one having. Could you explain?

File: Dockerfile [hunk 2]
```diff
@@ -7,9 +7,8 @@ RUN npm install
 RUN npm run build
 
 # Stage 2: Serve app with nginx
-FROM nginx:alpine
-COPY --from=build /app/deployment/nginx.conf /etc/nginx/nginx.conf
-COPY --from=build /app/dist/edc-demo-client /usr/share/nginx/html
-COPY --from=build /app/src/assets /usr/share/nginx/html/assets
+FROM nginx:alpine AS ngi
```
Comment (marcgs) [inline Dockerfile:10]: Is `AS ngi` required?

File: deployment/nginx.conf (hunk 1 above)
Comment (paullatzelsperger) [reply deployment/nginx.conf:3]: if i'm honest I took the "new" `nginx.conf` verbatim from [here](https://dev.to/oneofthedevs/docker-angular-nginx-37e4). it also seems to have gzip compression

File: Dockerfile (hunk 2 above)
Comment (paullatzelsperger) [reply Dockerfile:10]: nope, probably not. should I remove it?

File: deployment/nginx.conf (hunk 1 above)
Comment (marcgs) [reply deployment/nginx.conf:3]: ok lets just make sure that the reload issue is not present with this new version

**Pull request closed without merging**